website/snippets/available-tiers-iprestrictions.md

```diff
@@ -0,0 +1,7 @@
+:::info Limited to some dbt Cloud Enterprise tiers
```

Comment:
If we switch to partials, rename this file to
> website/snippets/_available-tiers-iprestrictions.md

Reply:
addressed in https://github.com/dbt-labs/docs.getdbt.com/pull/3616